### DISCUSSION TITLE
Which train frunctions do you whish?

### DISCUSSION THREAD

**Discussion by Jean28518** [Libre-TrainSim/Libre-TrainSim#25] [Ideas]
Please comment, what train functions you would like to see in Libre TrainSIm

**Comment by bodamat** [comment]
I love trains that carry people :)

**Comment by skyace65** [comment]
A non static camera so I can look around.

**Comment by Jean28518** [comment]
@skyace65 For 0.7 there will come an outer view where you can look around in the cabin itself its not possible at this time. I personally would do this at a later release ;)

**Comment by bodamat** [comment]
Would be a great turn on the train from buttons on a control panel just click them.

**Comment by Jean28518** [comment]
Should be the control panel at the HUD, or in 3D in the Cabin?
